app/assets/stylesheets/common/foundation/variables.scss

```diff
@@ -138,6 +138,7 @@ $box-shadow: (
   "menu-panel": 0 12px 12px rgba(0, 0, 0, 0.15),
   "card": 0 4px 14px rgba(0, 0, 0, 0.15),
   "dropdown": 0 2px 3px 0 rgba(0, 0, 0, 0.2),
+  "dropdown-lite": 0px 2px 12px 0px rgba(0, 0, 0, 0.1),
```

Comment:
Is there a place in core where we can use this? I'm a tiny bit hesitant to merge styles that have 0 uses in core.

Reply:
there is now. I just made it for the new autocomplete dropdowns